extensions/cornerstone/src/OHIFCornerstoneViewport.js

```diff
@@ -196,13 +168,10 @@ class OHIFCornerstoneViewport extends Component {
       );
     }
 
-    const sopClassUid = sopClassUids && sopClassUids[0];
-
     this.getViewportData(
       studies,
       studyInstanceUid,
       displaySetInstanceUid,
-      sopClassUid,
```

Comment:
Where is this defined? Did the API change?

Reply:
We originally had handlers for dragging SEG into cornerstone viewports. Since that approach is being changed, it is now unused so I removed it. Its only API internal to the component.